src/Window.cpp

```diff
@@ -1111,10 +1122,19 @@ QString Window::executeCommand(QString command) {
     }
     bool success = moveForward(numHalfSteps);
     return success ? "" : CRASH;
-  } else if (function == "turnRight" || function == "turnRight90") {
+  } else if (function == "runRight" ) {
+      turn(Movement::RUN_RIGHT);
+      return "";
+  } else if (function == "runLeft" ) {
+      turn(Movement::RUN_LEFT);
+      return "";
+  } else if (function == "turnBack" ) {
+      turn(Movement::TURN_RIGHT_180);
+      return "";
+  } else if (function == "turnRight" || function == "turnRight90" ) {
```

Comment:
Let's keep this PR strictly scoped to curve turns. Please split this into its own PR.